pkg/cable/libreswan/libreswan.go

```diff
@@ -131,6 +135,11 @@ func (i *libreswan) Init() error {
 		return fmt.Errorf("error starting Pluto: %v", err)
 	}
 
+	i.ipt, err = iptables.New()
```

Comment:
It would be better to include a wait for 5 seconds which will help in situations when lock cannot be immediately acquired.

You can do the following
`iptables.New(iptables.IPFamily(iptables.ProtocolIPv4), iptables.Timeout(5))`

Reply:
I am using the API from `github.com/submariner-io/submariner/pkg/iptables`. Seems like it is already being done there.